### PR TITLE
ROX-11441: Run auth E2E tests in pipeline.

### PR DESCRIPTION
**Pull request by dhaus67**
## Description

This PR will ensure the auth E2E tests will be run in pipeline, specifically within OSCI.

Besides the code changes, the credentials have been added to the vault used by OSCI.

### CODE REVIEW

**Comment by dhaus67** [comment]
Current dependencies on/for this PR:
* main
  * **PR #194** <a href="https://app.graphite.dev/github/pr/stackrox/acs-fleet-manager/194" target="_blank">Graphite</a> 
    * **PR #195** <a href="https://app.graphite.dev/github/pr/stackrox/acs-fleet-manager/195" target="_blank">Graphite</a> 
      * **PR #201** <a href="https://app.graphite.dev/github/pr/stackrox/acs-fleet-manager/201" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/acs-fleet-manager/201?utm_source=stack-comment).

File: .openshift-ci/tests/e2e.sh [hunk 1]
```diff
@@ -36,9 +36,30 @@ fi
 if [[ -n "$OPENSHIFT_CI" ]]; then
     log "Test suite is running in OpenShift CI"
     export GOARGS="-mod=mod" # For some reason we need this in the offical base images.
+
+    # When running in OpenShift CI, ensure we also run the auth E2E tests.
+    RUN_AUTH_E2E="true"
```
Comment (ivan-degtiarenko) [inline .openshift-ci/tests/e2e.sh:41]: nit: can be one line `export RUN_AUTH_E2E="true"`

**Review by ivan-degtiarenko — APPROVED** [review]
Looks good

**Review by mtodor — APPROVED** [review]

**Pull request merged**